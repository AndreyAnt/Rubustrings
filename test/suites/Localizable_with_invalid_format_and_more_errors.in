"Original string with one variable %@" = "Tranlation without variables";
"Invalid Format" = "Missing semicolon"

