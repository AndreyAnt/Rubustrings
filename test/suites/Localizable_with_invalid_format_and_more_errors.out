[34mProcessing file: "test/suites/Localizable_with_invalid_format_and_more_errors.in"[0m
test/suites/Localizable_with_invalid_format_and_more_errors.in:2: error: invalid format: "Invalid Format" = "Missing semicolon"
[31m[1m
Result: âœ˜ Some errors detected[0m[0m
